### PR TITLE
Adding final version of '01-fizzbuzz.js'

### PR DESCRIPTION
**Pull request by rk-pdx**
**Changes made** 

The fizzbuzz() function was added; if an integer between 1 and 100, inclusive, is divisibly by both 3 and 5 (i.e. no remainder), then "fizzbuzz" is printed to the console. If this condition is not met, control falls onto the next statement which checks if the integer is divisible by 3; if this is not met, then it checks if it is divisible by 5. If none of these conditions are met, then the integer is printed to console.

**Why I made these changes**

I structured my function in this way, because it made the most sense for control to "flow" from the most general to most specific conditions.

**What feedback I am seeking**

With this exercise, I aimed to get used to writing in Javascript again. I am looking for any general tips or other changes that could be made to the code to make it cleaner, or simply any Javascript-specific stylistic hints.

### CODE REVIEW

**Review by gleason9113 — APPROVED** [review]

File: hw1/01-fizzbuzz.js [hunk 1]
```diff
@@ -10,7 +10,21 @@ Use console.log() to write the proper output to the command line.
 **/
 
 const fizzbuzz = () => {
-  // Add your code here
+  for (let i = 0; i < 100; i++) {
+    curr = (i + 1)
```
Comment (smah29) [inline hw1/01-fizzbuzz.js]: Why do curr = (i+1), when you can run the loop from 1 to 100 directly, it increases the computation

File: hw1/01-fizzbuzz.js [hunk 2]
```diff
@@ -10,7 +10,21 @@ Use console.log() to write the proper output to the command line.
...
+      console.log("fizzbuzz")
+    }
+    else if (curr % 3 === 0) {
+      console.log("fizz")
+    }
+    else if (curr % 5 === 0) {
+      console.log("buzz")
+    }
+    else {
+      console.log(curr)
```
Comment (smah29) [inline hw1/01-fizzbuzz.js]: instead of multiple console.log use it at the end. You can define a variable, and for each case you can assign it.

**Review by smah29 — CHANGES_REQUESTED** [review]

File: hw1/01-fizzbuzz.js (hunk 1 above)
Comment (rk-pdx) [reply hw1/01-fizzbuzz.js]: Thank you for the feedback! I have implemented both suggestions. Please approve when you get the chance.

**Review by CzerPDX — APPROVED** [review]
Looks great! Nice cleanup to already straightforward code :)

**Pull request merged**